crates/win-api-wrappers/src/wts.rs

```diff
@@ -94,20 +89,16 @@ impl Drop for WTSVirtualChannel {
 }
 
 /// RAII wrapper for WTS memory handle.
-pub struct WTSMemory(*mut core::ffi::c_void);
+struct WTSMemory(*mut core::ffi::c_void);
 
 impl WTSMemory {
     /// # Safety
     /// `ptr` must be a valid pointer to a handle returned from `WTSVirtualChannelQuery`.
-    pub unsafe fn new(ptr: *mut core::ffi::c_void) -> Self {
+    pub(crate) unsafe fn new(ptr: *mut core::ffi::c_void) -> Self {
         Self(ptr)
     }
 
-    pub fn as_mut_ptr(&self) -> *mut core::ffi::c_void {
-        self.0
-    }
-
-    pub fn as_handle(&self) -> HANDLE {
+    pub(crate) fn as_handle(&self) -> HANDLE {
```

Comment:
Can probably remove `pub(crate)` since WTSMemory is not visible outside of the module?